integrationtests/mayavi/common.py

```diff
@@ -316,6 +316,9 @@ class TestCase(Mayavi):
     # `self.compare_image` was called in the test..
     offscreen = Bool(False)
 
+    # Profile for memory usage.
+    profile = Bool(False)
```

Comment:
The changes in the common.py do not perform memory profiling, but check for memory usage. Using the word `profile` is misleading.